solr/parser_test.go

```diff
@@ -21,7 +21,16 @@ func BenchmarkFireworkResultParser(b *testing.B) {
+
+	for i := 0; i < b.N; i++ {
+		parser := FireworkResultParser{}
```

Comment:
If possible please fix it. Thanks

On Wednesday 01 June 2016 05:46 PM, Nguyen Van Nhu wrote:
>
> In solr/parser_test.go 
> <https://github.com/vanng822/go-solr/pull/38#discussion_r65348573>:
>
> > @@ -21,7 +21,16 @@ func BenchmarkFireworkResultParser(b *testing.B) {
> >  		parser := FireworkResultParser{}
> >
> >  		parser.Parse(&data)
> > -		//fmt.Printf("%v\n", d)
> > +	}
> > +}
> > +
> > +func BenchmarkExtensiveResultParser(b *testing.B) {
> > +	data := []byte(`{"responseHeader":{"status":0,"QTime":0,"params":{"start":"0","q":"*:*","wt":"json","rows":"5"}},"response":{"numFound":12508577,"start":0,"docs":[{"id":"P27665908","p_name":"Archer Lady Case for Blackberry Q10","p_price_c":"175000.00,IDR","p_shop_name":"Nisa Toko","p_domain":"nisatoko","p_shop_sd_name":"Bekasi","p_key":"archer-lady-case-for-blackberry-q10","p_file_path":"product-1/2015/12/28/6208975","p_file_name":"6208975_3959f656-cb27-43df-83db-4b1c1455d3ce.jpg","p_id":27665908,"p_child_cat_id":69,"p_shop_id":733718,"p_condition":1,"p_server_id":1,"p_price":
>   175000.0
>   ,"p_dep_id":["65","69","66"],"p_best_match_dink":-288.0,"p_best_match_price":4.756962,"_version_":1521775745340276736},{"id":"P27665917","p_name":"shampo zoku penumbuh rambut hitam berkilau","p_price_c":"115000.00,IDR","p_shop_name":"penumbuh rambut sehat10","p_domain":"penumbuhrambut93","p_shop_sd_name":"Surabaya","p_key":"shampo-zoku-penumbuh-rambut-hitam-berkilau","p_file_path":"product-1/2015/12/28/6766154","p_file_name":"6766154_3344bc2d-8069-4ef6-adc7-7de90b6604f5.jpg","p_id":27665917,"p_child_cat_id":436,"p_shop_id":801110,"p_condition":1,"p_server_id":1,"p_price":115000.0,"p_dep_id":["433","61","436"],"p_best_match_din
>   k":
>   -288.0,"p_best_match_price":4.939302,"_version_":1521775745341325313},{"id":"P27665953","p_name":"Archer Lady Case for HTC One M7","p_price_c":"175000.00,IDR","p_shop_name":"Nisa Toko","p_domain":"nisatoko","p_shop_sd_name":"Bekasi","p_key":"archer-lady-case-for-htc-one-m7","p_file_path":"product-1/2015/12/28/6208975","p_file_name":"6208975_68882b13-3e53-431b-bd64-d2e518db03e9.jpg","p_id":27665953,"p_child_cat_id":69,"p_shop_id":733718,"p_condition":1,"p_server_id":1,"p_price":175000.0,"p_dep_id":["65","69","66"],"p_best_match_dink":-288.0,"p_best_match_price":4.756962,"_version_":1521775745341325315},{"id":"P27665898","p_name
>   ":&
>   quot;Archer Lady Case for Blackberry 9900 9980","p_price_c":"175000.00,IDR","p_shop_name":"Nisa Toko","p_domain":"nisatoko","p_shop_sd_name":"Bekasi","p_key":"archer-lady-case-for-blackberry-9900-9980","p_file_path":"product-1/2015/12/28/6208975","p_file_name":"6208975_b40a60e6-1909-4a13-9d49-f881d3a8c5da.jpg","p_id":27665898,"p_child_cat_id":69,"p_shop_id":733718,"p_condition":1,"p_server_id":1,"p_price":175000.0,"p_dep_id":["65","69","66"],"p_best_match_dink":-288.0,"p_best_match_price":4.756962,"_version_":1521775745342373888},{"id":"P27665910","p_name":"VAMPIRE SERUM 30ml ORIGINAL","p_price_c":"26600.00,IDR","p_shop_name":"Dokt
>   er Canti
>   kku","p_domain":"doktercantiku","p_shop_sd_name":"Jakarta","p_key":"vampire-serum-30ml-original","p_file_path":"product-1/2015/12/28/6634788","p_file_name":"6634788_e34791af-a170-48e8-b836-68da4c5d775a.jpg","p_id":27665910,"p_child_cat_id":598,"p_shop_id":785036,"p_condition":1,"p_server_id":1,"p_price":26600.0,"p_dep_id":["61","445","598"],"p_best_match_dink":-288.0,"p_best_match_price":5.5751185,"_version_":1521775745342373889}]}}`)
> > +
> > +	for i := 0; i < b.N; i++ {
> > +		parser := FireworkResultParser{}
>
> If you don't have time I can help you to fix this
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/vanng822/go-solr/pull/38/files/47fc826202fabe98d97262ab6de2e9fbced42913#r65348573>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ASRvIYqnh5I6nb1wgdrOgPkdf21Pt3J3ks5qHXgNgaJpZM4Ipd3V>.
>


-- 

------------------------------
IMPORTANT: This is an e-mail from HiFX IT Media Services Pvt. Ltd. Its 
content are confidential to the intended recipient. If you are not the 
intended recipient, be advised that you have received this e-mail in error 
and that any use, dissemination, forwarding, printing or copying of this 
e-mail is strictly prohibited. It may not be disclosed to or used by anyone 
other than its intended recipient, nor may it be copied in any way. If 
received in error, please email a reply to the sender, then delete it from 
your system. 

Although this e-mail has been scanned for viruses, HiFX cannot ultimately 
accept any responsibility for viruses and it is your responsibility to scan 
attachments (if any).

​
Before you print this email or attachments, please consider the negative 
environmental impacts associated with printing.